### PR TITLE
Sidekiq processing

### LINKED ISSUE #240: Switch from DelayedJob to sidekiq

### PR DESCRIPTION
**Pull request by hari816**
Fixed #240 #241

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -56,6 +46,8 @@ gem "autoprefixer-rails"
 # forms made easy for rails
 gem "simple_form"
 
+gem 'coffee-script'
```
[the remark this answers is not in the document]
Comment (hari816) [reply Gemfile]: While rendering layouts , I got error cannot load such file --coffee_script After upgrading the rails version , so I added the gem

File: app/mailers/mailer.rb [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 
   default from: Rails.application.secrets.mailer_default_from_email
 
   default_url_options[:host] = Rails.application.secrets.host
 
-  def contact_us_notification(contact)
-    @email = contact.email
-    @title = contact.title
-    @body  = contact.body
+  def contact_us_notification(title, email, body)
```
[the remark this answers is not in the document]
Comment (hari816) [reply app/mailers/mailer.rb]: yes contact_us_email makes sense

File: app/controllers/contacts_controller.rb [hunk 3]
```diff
@@ -2,10 +2,11 @@
 
 class ContactsController < ApplicationController
   def create
+    # binding.pry
     @contact = Contact.new(contact_params)
 
     if @contact.valid?
-      Mailer.delay.contact_us_notification(@contact)
```
[the remark this answers is not in the document]
Comment (hari816) [reply app/controllers/contacts_controller.rb]: Contact is just a model which validates the fields at Application level , But at DB level there are no migrations to store Contact , So create method wont work here

[the remark this answers is not in the document]
Comment (sunilkumarn) [reply app/controllers/contacts_controller.rb]: So, what are the plans here? Shouldn't we add them?

[the remark this answers is not in the document]
Comment (hari816) [reply app/controllers/contacts_controller.rb]: Not Required

File: test/workers/send_msg_of_user_worker_test.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```
Comment (sunilkumarn) [inline test/workers/send_msg_of_user_worker_test.rb]: Dont think we need this file anymore, right?

File: app/controllers/contacts_controller.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (sunilkumarn) [reply app/controllers/contacts_controller.rb]: Ok.

File: test/workers/send_msg_of_user_worker_test.rb (hunk 4 above)
Comment (hari816) [reply test/workers/send_msg_of_user_worker_test.rb]: Yes, will delete it

**Pull request merged**